src/stcal/resample/resample.py

```diff
@@ -1157,12 +1221,62 @@ def init_time_counters(self):
 
         self._measurement_time_success = []
 
-    def update_time(self, model):
-        """
-        A method called by the :py:meth:`~Resample.add_model` method to
+    def update_time(self, model, pixmap, iscale, weight_map,
```

Comment:
```suggestion
    def update_time(self, model, pixmap, iscale, weight_map,
```
Is the change to this function necessary anymore (since it's not used by roman or jwst)?

Reply:
I removed this. Can be restored later if needed. I think the proposed hook function can do what initial modification to `update_time` could do + more.